.github/workflows/ci.yml

```diff
@@ -89,5 +89,6 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
       - run: script/clone_all_rspec_repos
+      - run: bundle
```

Comment:
I had to restore the script to update bundler to get the main builds working 🤔 😬